### ISSUE TITLE
Video block: Remove unnecessary tooltip from Text Tracks button

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#66715]
### Description

In the Video block toolbar, the 'Text tracks' button has both visibel text as children and a `label` prop with the same value:

https://github.com/WordPress/gutenberg/blob/b992649af2753e818639fef6e38d34de63474822/packages/block-library/src/video/tracks-editor.js#L206-L214

The `lable` prop does two things:
- Renders an `aria-label` attribute with the same text that is already visible.
- Renders a tooltip when hovering or focusing the button with teh same text that is already visible.

Both the aria-label and tooltup are unnecessary and should be removed.

![Image](https://github.com/user-attachments/assets/765b436c-c920-4fa5-b45e-2268a389d760)

### Step-by-step reproduction instructions

- Add a Video block.
- Hover or focus the 'Text tracks' button.
- Observe a tooltip appears.
- Observe the tooltip has the same text as the button visible text.
- Inspect the source and obseve the button has an aria-lable attribute with the same text that is already visible.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
